client/src/pages/create-database.tsx

```diff
@@ -104,118 +126,211 @@ export const CreateDatabase = () => {
 
   const isPluginInstalled = data?.isPluginInstalled.isPluginInstalled;
 
+  const handleNext = () => {
+    try {
+      subscribeToMore({
```

Comment:
@pradel this works, but would like to hear some feedback whether it's a good solution. From what I understand it subscribes to query and updates the result if there's any update. 

Is that correct and does application of this `subscribeToMore` make sense in this case?

Reply:
@Akirtovskis I am not sure what you are tried to achieve here. If you are just trying to get the database id why don't you send it from the server via the `end` event?

Reply:
@pradel I was indeed just trying to get the id of db and thanks for the elegant & simple suggestion.  Fixed now.